extensions/rich_text_components/Image/directives/ImageDirective.js

```diff
@@ -21,11 +21,14 @@
  */
 oppia.directive('oppiaNoninteractiveImage', [
   '$rootScope', '$sce', 'HtmlEscaperService', 'explorationContextService',
-  function($rootScope, $sce, HtmlEscaperService, explorationContextService) {
+  'UrlInterpolationService',
+  function($rootScope, $sce, HtmlEscaperService, explorationContextService,
+  UrlInterpolationService) {
```

Comment:
Please break at '(' if the contents of (...) extend over multiple lines.

Reply:
Done.